### PR TITLE
chore: remove celo dependencies from e2e tests

### PR DESCRIPTION
**Pull request by kathaypacific**
### Description

This PR removes usages of all `@celo` dependencies from the e2e folder.

### Related issues

- Related to RET-1173

### Network scalability

If a new NetworkId and/or Network are added in the future, the changes in this PR will:

- [ ] Continue to work without code changes, OR trigger a compilation error (guaranteeing we find it when a new network is added)

### CODE REVIEW

File: e2e/src/utils/utils.js [hunk 1]
```diff
@@ -424,19 +428,45 @@ export async function completeProtectWalletScreen() {
...
+    data: encodeFunctionData({
+      abi: erc20.abi,
+      functionName: 'transfer',
+      args: [recipientAddress, fundingAmount],
+    }),
+  })
+
+  const client = createPublicClient({
+    chain: celoAlfajores,
+    transport: http(ALFAJORES_FORNO_URL),
```
Comment (jeanregisser) [inline e2e/src/utils/utils.js:466]: nit:
```suggestion
    transport: http(),
```

File: e2e/src/utils/utils.js [hunk 2]
```diff
@@ -424,19 +428,45 @@ export async function completeProtectWalletScreen() {
...
-    .transfer(recipientAddress, amountWei.toString())
-    .sendAndWaitForReceipt({ from: senderAddress })
+
+  const account = privateKeyToAccount(senderPrivateKey)
+  const senderAddress = account.address
+  const walletClient = createWalletClient({
+    account,
+    chain: celoAlfajores,
+    transport: http(ALFAJORES_FORNO_URL),
+  })
```
Comment (jeanregisser) [inline e2e/src/utils/utils.js:451]: nit: we can create a client that can do both wallet and public actions:

```suggestion
import { createWalletClient, http, publicActions } from 'viem'
 
const client = createWalletClient({ 
  account,
  chain: celoAlfajores,
  transport: http()
}).extend(publicActions) 
```

File: src/viem/getLockableWallet.test.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { normalizeAddress } from '@celo/utils/lib/address'
-import erc20 from 'src/abis/IERC20.json'
+import erc20 from 'src/abis/IERC20'
```
Comment (jeanregisser) [inline src/viem/getLockableWallet.test.ts:2]: nit:
```suggestion
import { erc20Abi } from 'viem'
```

**Review by jeanregisser — APPROVED** [review]
Awesome! 🚀

File: e2e/src/usecases/WalletConnectV2.js [hunk 4]
```diff
@@ -91,6 +91,11 @@ export default WalletConnect = () => {
   let walletConnectClient, pairingUrl, core
   let intervalsToClear = []
 
+  const client = createPublicClient({
+    chain: celoAlfajores,
+    transport: http(transportUrl),
```
Comment (bakoushin) [inline e2e/src/usecases/WalletConnectV2.js:96]: as Jean mentioned, we can omit it

File: e2e/src/usecases/WalletConnectV2.js [hunk 5]
```diff
@@ -17,17 +19,15 @@ import { enterPinUiIfNecessary, navigateToSettings, sleep } from '../utils/utils
...
+const transportUrl = process.env.FORNO_URL || 'https://alfajores-forno.celo-testnet.org'
 
 const walletAddress = (
   process.env.E2E_WALLET_ADDRESS || '0x6131a6d616a4be3737b38988847270a64bc10caa'
 ).toLowerCase()
 
 async function formatTestTransaction(address) {
   try {
-    const response = await fetch(kitUrl, {
+    const response = await fetch(transportUrl, {
```
Comment (bakoushin) [inline e2e/src/usecases/WalletConnectV2.js:30]: i wonder if we can use [getBlockTransactionCount](https://viem.sh/docs/actions/public/getBlockTransactionCount#getblocktransactioncount) instead? it is less code.

**Review by bakoushin — APPROVED** [review]

**Pull request merged**